src/python_spy.rs

```diff
@@ -17,6 +17,400 @@ use remoteprocess::{Process, ProcessMemory, Pid, Tid};
 use proc_maps::{get_process_maps, MapRange};
 
 
+#[cfg(all(target_os = "linux", feature = "trace_syscalls"))]
+lazy_static! {
+    static ref SYSCALLS: HashMap<u32, &'static str> = {
+        let mut m = HashMap::new();
```

Comment:
How was the list of syscalls generated? What did you use as a datasource here?

Reply:
I extracted it from https://github.com/torvalds/linux/blob/9b76d71fa8be8c52dbc855ab516754f0c93e2980/arch/x86/entry/syscalls/syscall_64.tbl, but maybe it makes sense to support multiple architectures here or fail if the architecture is not x86_64

Reply:
I changed this to use the `syscalls` crate to determine the name of the syscalls